<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDL HELPER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="classlabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	
border-image: url(:/icon/kebiao.png);
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(255,255, 255); 
	background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="arrangelabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	
border-image: url(:/icon/todayq.png);
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(255,255, 255); 
	background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="weekreportpushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(17,17,17);
	
	border-image: url(:/icon/weekre.png);
	
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
	font: 17pt &quot;Heiti SC&quot;;
color:white
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="classpushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-image: url(:/icon/class.png);
	
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
	font: 17pt &quot;Heiti SC&quot;;
color:white
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>1031</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel { 
background-color: rgb(139 ,0, 18); }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
	border-image: url(:/ne/22.png);

	background-color: rgb(139, 0, 18);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-radius:20px;
	background-color: rgb(139, 0, 18);
}</string>
    </property>
    <widget class="QTableWidget" name="arrangetableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	background-color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);
 font-size: 20pt;
}
QHeaderView::section {
background-color: rgb(139,0,18);
      color: white;          
      padding: 4px;        
     border: 1px solid rgb(139,0,18);
      min-height: 25px;        
    }
</string>
     </property>
     <column>
      <property name="text">
       <string>待办任务名称</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>截止日期</string>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>重要程度</string>
      </property>
      <property name="font">
       <font>
        <family>.Apple Color Emoji UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="aboutpushButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 24pt &quot;Times New Roman&quot;;
border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
color:white;
}

</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>311</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
border-radius:20px;
	background-color: rgb(139, 0, 18);
}</string>
    </property>
    <widget class="QTableWidget" name="classtableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	background-color: rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
gridline-color:rgb(139,0,18);
 font-size: 17pt;
}
QHeaderView::section {
background-color: rgb(139,0,18);
      color: white;          
      padding: 4px;        
     border: 1px solid rgb(139,0,18);
      max-height: 25px;        
	font: 20pt &quot;Heiti SC&quot;;
    }
</string>
     </property>
     <row>
      <property name="text">
       <string>1.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9.</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10.</string>
      </property>
     </row>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>课程名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>信息</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>660</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(17,17,17);
	
	
	font: 48pt &quot;Heiti SC&quot;;
	
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
	
	font: 700 72pt &quot;Arial&quot;;
color:white
}
QPushButton:hover { 
background-color: rgb(167, 42, 43);}
QPushButton:hovered { 
background-color: rgb(116, 255, 63);}

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adjustbutton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>660</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(17,17,17);
	
	
	font: 48pt &quot;Heiti SC&quot;;
	
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
	
	font: 700 78pt &quot;Arial&quot;;
color:white
}
QPushButton:hover { 
background-color: rgb(167, 42, 43);}

</string>
    </property>
    <property name="text">
     <string>✎</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletebutton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>660</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(17,17,17);
	
	
	font: 48pt &quot;Heiti SC&quot;;
	
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
	
	font: 700 72pt &quot;Arial&quot;;
color:white
}

QPushButton:hover { 
background-color: rgb(167, 42, 43);}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 0, 18);
border-image: url(:/icon/llooggoo.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="notepushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-image: url(:/icon/note.png);
	
	border-width: 1px;
	border-radius: 10px;
	border-color: rgb(139 ,0, 18); 
	background-color: rgb(139, 0, 18);
	font: 17pt &quot;Heiti SC&quot;;
color:white
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>classlabel</zorder>
   <zorder>widget_2</zorder>
   <zorder>label</zorder>
   <zorder>arrangelabel</zorder>
   <zorder>weekreportpushButton</zorder>
   <zorder>classpushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>widget</zorder>
   <zorder>aboutpushButton</zorder>
   <zorder>addButton</zorder>
   <zorder>adjustbutton</zorder>
   <zorder>deletebutton</zorder>
   <zorder>label_3</zorder>
   <zorder>notepushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
